test/rules/type-literal-delimiter/test.ts.lint

```diff
@@ -0,0 +1,28 @@
+type T = {
```

Comment:
Not currently, no. I think that would be more appropriate for the `whitespace` rule.